bin/es-docker

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Run Elasticsearch and allow setting default settings via env vars
```

Comment:
Given that we control the command we're using to call ES along with its environment, we don't really need this shell script. We can include its effects entirely in the `docker-compose.yml`'s `command` section.